tests/unit/t_header.c

```diff
@@ -152,6 +152,10 @@ void set_aad_tbl(struct aws_cryptosdk_hdr *hdr, struct aws_cryptosdk_hdr_aad *aa
 }
 
 static struct aws_cryptosdk_hdr test_header_1_hdr() {
+    struct aws_allocator *allocator = aws_default_allocator();
+    if(!allocator){
+        abort();
```

Comment:
We generally donot use `abort()` within our functions for memory allocation errors, we release all prior memory allocated, if any using `aws_mem_release()`  first and then return NULL instead.

Reply:
removed